src/components/shell/shell.js

```diff
@@ -313,9 +327,19 @@ class Shell extends BaseComponent {
     if (this.props.seekbarDraggingActive) playerClasses.push(style.hover);
     if (this.props.fullscreen) playerClasses.push(style.fullscreen);
     if (this.props.playlist) playerClasses.push(style.playlist);
-    if (this.props.playerClientRect && this.props.playerClientRect.width <= 480) playerClasses.push(style.sizeSm);
-    else if (this.props.playerClientRect && this.props.playerClientRect.width <= 768) playerClasses.push(style.sizeMd);
-    else if (this.props.playerClientRect && this.props.playerClientRect.width <= 1024) playerClasses.push(style.sizeLg);
+    if (this.props.playerClientRect && this.props.playerClientRect.width <= PLAYER_BREAK_POINTS.SMALL) {
+      playerClasses.push(style.sizeSm);
+      this.props.updatePlayerSize(PLAYER_SIZE.SMALL);
+    } else if (this.props.playerClientRect && this.props.playerClientRect.width <= PLAYER_BREAK_POINTS.MEDIUM) {
+      playerClasses.push(style.sizeMd);
+      this.props.updatePlayerSize(PLAYER_SIZE.MEDIUM);
+    } else if (this.props.playerClientRect && this.props.playerClientRect.width <= PLAYER_BREAK_POINTS.LARGE) {
+      playerClasses.push(style.sizeLg);
+      this.props.updatePlayerSize(PLAYER_SIZE.LARGE);
+    } else {
+      playerClasses.push(style.sizeLg);
```

Comment:
remove. this breaks the default style. 
or add `sizeXlg` (and add a black style, otherwise it won't be added)

Reply:
removed